mod_reforged/hooks/entity/tactical/actor.nut

```diff
@@ -13,6 +13,10 @@
 	{
 		__original();
 		this.m.RF_DamageReceived = { Total = 0.0 };
+		if (::World.Assets.getCombatDifficulty() == ::Const.Difficulty.Easy && this.getType() != ::Const.EntityType.Player)
+		{
+			this.m.BaseProperties.DamageTotalMult = 0.9;
+		}
```

Comment:
All the changes in this PR also need to be somehow reflected in the tooltips of the difficulty settings.